src/validators/satelliteNameValidator.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response, response } from 'express'
+
+export const satelliteNameRouteValidation = (req: Request, res: Response, next: CallableFunction): void | ReturnType<typeof response.sendStatus> => {
```

Comment:
Change `CallableFunction` to `NextFunction` (import from express), and if you like, `void` to `ReturnType<NextFunction>`.